src/app/shared/components/filters-list/city-filter/city-filter.component.html

```diff
@@ -23,24 +24,24 @@
     </mat-option>
   </mat-autocomplete>
   <p *ngIf="settlement?.fullName" class="additional-text">
-    {{ settlement.fullName | slice: 0:30 }}
-    <span *ngIf="settlement.fullName.length > 30">...</span>
+    {{ settlement.fullName | slice: 0:25 }}
+    <span *ngIf="settlement.fullName.length > 25">...</span>
   </p>
 </mat-form-field>
 
 <!-- CITY CONFIRMATION -->
 <div class="confirmation">
   <div class="city-confirmation" *ngIf="isDispalyed && !(isConfirmCity$ | async)">
     <div fxLayout="row" fxLayoutAlign="space-between start">
-      <p>
+      <p fxFlex="93%">
         Ви знаходитесь в населеному пункті <span>{{ settlement.settlement }}</span
         >?
       </p>
-      <mat-icon (click)="isDispalyed = false">clear</mat-icon>
+      <mat-icon (click)="isDispalyed = false" fxFlex="7%">clear</mat-icon>
     </div>
     <div fxLayout="row" fxLayoutAlign="space-between center" class="city-confirmation_button">
       <button mat-button class="button" (click)="confirmCity()">Підтвердити</button>
-      <button mat-button class="button button-change" (click)="changeCity()">Змінити</button>
+      <button mat-button class="button button-change" (click)="changeCity(searchInput)">Змінити</button>
```

Comment:
I think you can get access to the search input via ViewCHild and get rid of passing the value